README.md

```diff
@@ -39,17 +37,15 @@ This repository contains examples of various algorithms written on different pro
 | [Binary Search Tree](https://en.wikipedia.org/wiki/Binary_search_tree)                          |                                       | [:octocat:](binary_search_tree/Cpp)                                       |                                       |                                       |
 | [Fenwick Tree](https://en.wikipedia.org/wiki/Fenwick_tree)                                      |                                       |                                       | [:octocat:](fenwick_tree/java)        | [:octocat:](fenwick_tree/Python)      |
 
-
 ## Sample Run
 
-| Language        | Steps                                                                  |
-| --------------- | ---------------------------------------------------------------------- |
-| JavaScript      | <pre>node [filename.js]</pre>                                          |
-| Python          | <pre>python [filename.py]</pre>                                        |
-| C               | <pre>gcc [filename.c]<br>./a.out  # unix<br>a.exe  # windows</pre>     |
-| CPP             | <pre>g++ [filename.cpp]<br>./a.out # unix<br>a.exe # windows</pre>     |
-| Java            | <pre>javac [filename.java]<br>java [filename]</pre>                    |
-
+| Language        | Steps                                                                    |
+| --------------- | ------------------------------------------------------------------------ |
+| JavaScript      | ```node [filename.js]```                                                 |
+| Python          | ```python [filename.py]```                                               |
+| C               | ```gcc [filename.c]```<br>```./a.out  # unix```<br>```a.exe  # windows```|
+| CPP             | ```g++ [filename.cpp]```<br>```./a.out # unix```<br>```a.exe # windows```|
+| Java            | ```javac [filename.java]```<br>```java [filename]```                     |
```

Comment:
These changes needs to be ignored as it is being used as HTML tags and then being used for the website https://nitskmos.github.io/Algorithms/.